docs/section4/4-16.md

```diff
@@ -0,0 +1,50 @@
+- Combine list of hvac systems with virtual heating and/or cooling: `HVAC_systems_virtual_list_p = HVAC_systems_virtual_cooling_list_p + HVAC_systems_virtual_heating_list_p`
+- Eliminate duplicate HVAC systems: `HVAC_systems_virtual_list_p = list(set(HVAC_systems_virtual_list_p))`  
+Create a list of the zones served by the HVAC systems in which virtual heating and/or cooling has been modeled in the proposed RMR
+- Get dictionary with list of zones associated with each HVAC system: `dict_hvac_sys_zones_served_p = get_hvac_zone_list_w_area(P_RMR)`
+- Loop through the HVAC systems with virtual heating and/or cooling and add to list of zones with virtual heating and/or cooling: `For hvac_p in HVAC_systems_virtual_list_p:`
+    - Get list of zones associated with the hvac system and add to list of zones with virtual heating and/or cooling: `zones_virtual_heating_cooling_list = zones_virtual_heating_cooling_list.append(list(dict_hvac_sys_zones_served_p[hvac_p.id]["Zone_List"].values()))`
+- Eliminate duplicate zones from the list: `zones_virtual_heating_cooling_list = list(set(zones_virtual_heating_cooling_list))`
+- Loop through each zone in the zones_virtual_heating_cooling_list and obtain the hvac system serving the zone in the B_RMR and add to the list of HVAC systems applicable to this check: `For zone_p in zones_virtual_heating_cooling_list:`
+    - Get analogous zone in the B_RMR: `zone_b = match_data_element(B_RMR, Zone,zone_p.id)`  
+    - Get list of hvac systems serving the zone and add to list of applicable hvac systems: `applicable_hvac_with_virtual_heating_cooling = applicable_hvac_with_virtual_heating_cooling.append(get_list_hvac_systems_associated_with_zone(B_RMR, zone_b.id))`  
+- Eliminate duplicates in list of hvac systems: `applicable_hvac_with_virtual_heating_cooling = list(set(applicable_hvac_with_virtual_heating_cooling))`
+- For each HeatingVentilationAirconditioningSystem in the B_RMR check if this check is applicable: `for each hvac_b in B_RMR..HeatingVentilationAirConditioningSystem:`
+    - Check if hvac_b is in the list of applicable systems, if it is found then set applicability flag to true: `if hvac_b in applicable_hvac_with_virtual_heating_cooling: rule_applicability_flag = TRUE`   
+    - If rule_applicability_flag = TRUE for hvac_b: `if rule_applicability_flag == TRUE:`  
+        - Get operation during occupied periods: `operation_during_occupied_b = hvac_b.fan_systems[0].operation_during_occupied`  
+
+        **Rule Assertion:**  
+        - Case 1: For each applicable hvac system: `outcome == "UNDETERMINED" and raise_message "Check that heating or cooling system that is only being simulated in the P_RMR to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d are modeled in the B_RMR to be cycled ON and OFF to meet heating and cooling loads during occupied hours."`
```

Comment:
@mkarpman since we're giving an undetermined outcome, I don't see the point of doing all of this evaluation. Maybe this should be a non-mandatory rule and the applicability could be determined based on the presented of a virtual H or C system in the P RMR.

Reply:
@mkarpman and @claperle I am also unclear on why we're not doing the additional step of checking if operation_during_occupied_b  = cycling. Looks like we could do that and give a pass outcome?

Let me know what I'm missing.

Reply:
as discussed, will leave this RDS as is. For incorporating the operation_during_occupied_b in the outcome message we could simply revise the outcome message to say 
Check that heating or cooling system that is only being simulated in the P_RMR to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d are modeled in the B_RMR to be cycled ON and OFF to meet heating and cooling loads during occupied hours. The fan operation during occupied for this HVAC system is 'operation_during_occupied_b'